RemoteHealthcare/SimulatorBike.cs

```diff
@@ -25,22 +18,16 @@ public void SetResistance(byte resistance)
         public void SetAirResistance(byte airResistanceCoefficient, byte windspeed, byte draftingFactor)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)